working/wildcards/feature-specification.md

```diff
@@ -317,37 +317,54 @@ class C {
 
 ### Super parameters
 
-An occurrence of `super._` as a declaration of a formal parameter in a
-constructor is a compile-time error. This error also occurs in the case
-where the super parameter has an explicitly declared type and/or default
-value.
+The positional formal parameter `super._` is still allowed in generative
```

Comment:
Pedantic: ... in non-redirecting generative constructors.

(Everybody knows what you mean, most people don't even remember that redirecting generative constructors exist.)